lib/openstax/rescue_from/logger.rb

```diff
@@ -0,0 +1,31 @@
+module OpenStax
+  module RescueFrom
+    class Logger
+      attr_reader :wrapped
+
+      def initialize(wrapped)
+        @wrapped = wrapped
+      end
+
+      def record_system_error!
```

Comment:
Cool, sounds good. What you saw were my interpreted assumptions from the original initializer so more information always helps. Thanks